messaging/src/main/java/org/axonframework/commandhandling/CommandExecutionException.java

```diff
@@ -39,4 +39,17 @@
     public CommandExecutionException(String message, Throwable cause) {
         super(message, cause);
     }
+
+    /**
+     * Initializes the exception with given {@code message}, {@code cause} and an object providing application-specific
+     * {@code details}.
+     *
+     * @param message The message describing the exception
+     * @param cause   The cause of the exception
+     * @param details An object providing more error details (may be {@code null}
+     */
+    public CommandExecutionException(String message, Throwable cause, Object details) {
```

Comment:
Just for my understanding, this exception isn't being thrown by the framework itself, right?

If I understand correctly, this constructor is added for the purpose that a user may throw it, so that it'll be de-/serialized through an Axon Server instance to get back to the original dispatcher of the given command which failed execution.